make/lib/Lib-jdk.hotspot.agent.gmk

```diff
@@ -80,6 +80,8 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBSA, \
     LIBS_windows := dbgeng.lib, \
 ))
 
-TARGETS += $(BUILD_LIBSA)
+ifneq ($(OPENJDK_TARGET_OS)-$(OPENJDK_TARGET_CPU), bsd-arm)
+  TARGETS += $(BUILD_LIBSA)
+endif
```

Comment:
Is this a lack of porting the serviceability agent to arm32 or does it not work on arm32 in general (e.g. Linux also doesn't support it)

Reply:
I don't remember, I'll try to build it again.

Reply:
It doesn't seem to be implemented in linux, I can't find anything in  `src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/debugger/linux` or in `src/jdk.hotspot.agent/linux/native/libsaproc/LinuxDebuggerLocal.c`

Reply:
I downloaded the AdoptOpenJDK distribution for arm32 and it does contain libsaproc.so, so it seems like Linux is building the pieces for this even though it may not have arch specific files for it.  Does it error out building on FreeBSD?